src/test/java/net/pms/test/formats/FormatFactoryTest.java

```diff
@@ -165,6 +165,7 @@ public final void testFormatRetrieval() {
 		testSingleFormat("test.wmv", "MPG", Format.VIDEO);
 		testSingleFormat("test.wtv", "MPG", Format.VIDEO);
 		testSingleFormat("test.mpa", "MPGAudio", Format.AUDIO);
+		testSingleFormat("test.mp1", "MPGAudio", Format.AUDIO);
```

Comment:
@Sami32 Sorry. You (should) know that I don't like using cryptic codes where descriptive names could be used. In this case though, I actually agree because it's more coherent with the way the other identifiers. Ideally though, I'd change them all to descriptive names, but that's something I can do on "my own" project.

I've edited my answer, please go ahead and do it.